### ISSUE TITLE
[BUG] {serious - 2.06 unusable} Editing sluggish and items disappearing

### ISSUE THREAD

**Issue by rogercreagh** [TechbeeAT/jtxBoard#945]
**Describe the bug**
Two parts. Firstly while creating a new Journal entry the UI becomes very sluggish - editing description lags up to 5 secs behind keystrokes before the appear on screen - sometimes not all will appear, or will appear in 2 or more batches of a few separated by a few seconds. This used to happen occasionally in 2.05

Secondly after struggling with this for a bit including saving the entry, and reopening it, and exiting jtxB and restarting it, After a while it froze with the error message that you cant escape from about the item having been deleted in the background  (see previous issue that we thought had been solved). Had to force closure of jtxB

When I reopened jtxB next time the three previous entries had also disappeared. I instantly turned off wifi and checked on the server and all entries were still there including the new one which didn't have the last few keystrokes.

The last few keystrokes may have included a backspace delete - I wonder if these had been used to delete entries when jtxB returned to the list - incredibly dangerous behaviour.

Fortunately I have a backup from the server and another device also on 2.06 but I now daren't renable wifi on my primary device for fear it will delete the entries on the server when davx fires up oruse the other device which is also on 2.06. WTF should I do??? How do I downgraded to a working version without loosing anything from server?

**To Reproduce**
Steps to reproduce the behaviour:
1. upgrade to 2.06
2. Create a new journal entry and enter a description
3. Observe sluggish behaviour while editing
4. Delete last few chars and save & exit editing and go back to list or carry on till it crashes with the message about item having been deleted in background
5. Restart - a number of recent entries will have been deleted

**Expected behavior**
Working normally as it was in 2.02

**Device and version:**
 - Device: primary Boox eInk tablet, secondary Blackview 7100 phone
 - Android version 11
 - jtx Board version: 2.06.0
 - Downloaded from: f-Droid

**Comment by rogercreagh** [comment]
I can confirm the sluggish editing is also happening on a second device using 2.06. It starts after you have typed a few lines including jumping back to edit within the line.

**Comment by patrickunterwegs** [comment]
Hi @rogercreagh , 
the problem is that nothing has changed in the edit view that could have such a serious impact. Can you tell if the sync is running while editing gets sluggish?

The deleted entries are just a mystery. I mean, basically it's the sync algorithm from DAVx5 that does the things here. Do you get any errors from DAVx5?

About downgrading: You could only delete the app and download another version from GitHub or f-Droid. Just make sure you backup data if you have local entries as well.

**Comment by rogercreagh** [comment]
At the moment I'm not convinced that sync is running when it gets sluggish as the problem goes on much longer that a sync normally takes - unless Davx which is set to 30mins + immediate on local changes is detecting every update of the edit screen as a local change and initiating a sync.

No errors from Davx

I ensured the safety of the missing items that were present on the secondary device by saving a trivial change to them on that device and syncing so that there was a later timestamp then the missing ones on the primary device. Then sync'd the primary device and they came back.

Since only a few people are reporting the sluggish editing I'm thinking it must be something to do with using a specific editor - in my case OpenBoard v1.4.5 installed from Fdroid (playstore is still on 1.4.4 - unusual for it to be behind Fdroid)

For now I have set Davx to manual sync only (there is no option for (eg) 30mins and not also on local changes) and will report back.
